src/wilcoxon.jl

```diff
@@ -179,6 +182,43 @@ function pvalue(x::ApproximateSignedRankTest; tail=:both)
     end
 end
 
+ci(x::ApproximateSignedRankTest, alpha::Float64=0.05; tail=:both) = calculate_ci(x.vals, alpha; tail=tail)
+
+function calculate_ci(x::AbstractVector, alpha::Float64=0.05; tail=:both)
+    check_alpha(alpha)
+
+    if tail == :both
+        c = 1 - alpha
+    else
+        c = 1 - 2 * alpha
+    end
+    n = length(x)
+    m = convert(Int, n * (n + 1) / 2)
```

Comment:
No need to promote back and forth to `Float64`. You can write `div(n * (n - 1), 2)`.